### ISSUE TITLE
HIGH PRIORITY - [Screen Readers -View Details of APIs - Authorize]: Label 'Value' is not associated with their respective 'Edit fields'.

### ISSUE THREAD

**Issue by colsonrice** [pivotal-cf/service-instance-logs-cli-plugin#26]
https://jira.eng.vmware.com/browse/VPAT-3986
